nixos/modules/services/web-apps/cryptpad.nix

```diff
@@ -0,0 +1,262 @@
+{ config, lib, pkgs, ... }:
```

Comment:
Can you pass each new files from this PR through `nixfmt-rfc-style` ?

Reply:
Done. I also don't normally run a linter (too many of them and no clue what's standard) but tried statix which didn't complain

Reply:
The official linter is now `nixfmt-rfc-style`, this is why I require new files to be formatted with it.

Reply:
Sorry I meant lint as extra checks here (e.g. checking for unused inputs and other common mistakes), I don't expect a tool named nixfmt to do this kind of work.

I ran nixfmt-rfc-style first and updated this branch before my previous comment

Reply:
Indeed, `nixfmt-rfc-style` doesn't do any kind of static analysis or similar. It's just for the code style. As of today, I don't know any kind of tool that does that, and I hope it will be available one day. Something similar to `ruff` for python would be very nice.

Reply:
I think [statix](https://github.com/oppiliappan/statix) compares fairly well to linters like `ruff`...

Reply:
Right, I had heard about statix and that's why I had tried it earlier, but looking further it doesn't look like it checks for unused lambda patterns like this - only some antipatterns like `foo = lib.foo;` instead of `inherit (lib) foo;`.

A second look made me found [deadnix](https://github.com/astro/deadnix) and [nil](https://github.com/oxalica/nil); which both found I had one in the cryptpad package (fetchpatch) I've just removed now.

So, yeah, still too many tools; but I've tried setting up some hooks and will hopefully not need reminding anymore now that's done... Anyway, that's getting a bit off topic. Thanks for the reminder :)

Reply:
Oh yeah, sorry did not see that. I always find statix+deadnix to make a great combination for Nix linting...